buildscripts/Castle.Windsor.nuspec

```diff
@@ -29,14 +29,14 @@
   </metadata>
   <files>
     <file src="BreakingChanges.txt" />
-    <file src="Changes.txt" />
-    <file src="License.txt" />
+    <file src="CHANGELOG.md" />
+    <file src="LICENSE" />
     <file src="build/NET35/NET35-Release/pkg/bin/Castle.Windsor.???" target="lib/net35" />
     <file src="build/NET40/NET40-Release/pkg/bin/Castle.Windsor.???" target="lib/net40" />
     <file src="build/NET40CP/NET40CP-Release/pkg/bin/Castle.Windsor.???" target="lib/net40-client" />
     <file src="build/NET45/NET45-Release/pkg/bin/Castle.Windsor.???" target="lib/net45" />
     <file src="buildscripts/ASL - Apache Software Foundation License.txt" />
-    <file src="buildscripts/readme.txt" />
+    <file src="buildscripts/README.md" />
```

Comment:
I'm pretty sure this little `readme.txt` file is explicitly for display when the Windsor package is installed by NuGet as it'll get displayed:
- https://docs.microsoft.com/en-us/nuget/create-packages/creating-a-package#adding-a-readme-and-other-files